features.md

```diff
@@ -16,3 +16,14 @@ As a teeworlds user, I would like to respawn when the enemy that killed me is ki
 As a tee worlds user, I would like be able to win the game when all enemy players have been eliminated so that this game mode works like dodgeball really does.
 * Detect number of enemies on each death. If there is one team with nobody left, then the other team wins.
 
+
+## Requirements:
```

Comment:
Would it be better if the requirements were labeled something like 1.0, 1.1, 1.2. etc. instead of just all being 1?

Reply:
If there were other major sub-tasks, I would label them that way, but since this particular project is just adding the one game mode, I felt that just incrementing by 1 as whole numbers was sufficient.

I'll keep this in mind for the future though!